apps/web/src/components/createFormTemplate/CreateFormTemplateModal.tsx

```diff
@@ -84,25 +86,37 @@ export const CreateFormTemplateModal = ({
     setSignatureFields(tempSignatureFields);
   };
 
-  const submitFormTemplate = async () =>
+  const submitFormTemplate = async () => {
+    if (!pdfFile) {
+      throw new Error('No PDF file uploaded');
```

Comment:
Also is it better to throw a toast error here, or should we change this logic to something like the form signature fields where this needs field to present otherwise the create template form is disabled until then?

Reply:
it should already be disabled as of now I think